db/models/api.model.js

```diff
@@ -111,6 +111,18 @@ function deleteComment(comment_id) {
 
 }
 
+function readUsers() {
+  return db.query(`
+    SELECT * FROM users;
+    `)
+    .then(({rows}) => {
+      if(rows.length === 0){
+        return Promise.reject({status: 404, msg: "Users not found"})
+      }
```

Comment:
Do you ever end up in this if block? If it's not tested I would remove.

The only way this fails is if the database is down - it should default to a 500 (last error handler you should have in app.js)